### PR TITLE
fix: not throw warnings when using implicit fallback

### PR DESCRIPTION
**Pull request by ShinnosukeKomiya**
resolves https://github.com/intlify/vue-i18n-next/issues/1768

### CODE REVIEW

File: packages/vue-i18n-core/test/issues.test.ts
Comment (kazupon) [inline packages/vue-i18n-core/test/issues.test.ts:1]: I would also like to see additional tests for complex fallbacks using decision maps. 🙏 
https://vue-i18n.intlify.dev/guide/essentials/fallback.html#explicit-fallback-with-decision-maps

**Review by kazupon — CHANGES_REQUESTED** [review]
Thank you so much!
Great job!

I've just reviewed your PR, and added comment.
Please check it out.

File: packages/vue-i18n-core/test/issues.test.ts
Comment (ShinnosukeKomiya) [reply packages/vue-i18n-core/test/issues.test.ts:1]: Thank you for your review!
I have added tests for complex fallbacks using decision maps, so I hope you'll check it out 🙏

**Review by kazupon — APPROVED** [review]

**Comment by kazupon** [comment]
@ShinnosukeKomiya 
Thank you!
I've just checked your test codes!
LGTM!

**Pull request merged**